site/content/docs/5.1/examples/list-groups/list-groups.css

```diff
@@ -1,16 +1,3 @@
-.b-example-divider {
-  height: 3rem;
-  background-color: rgba(0, 0, 0, .1);
-  border: solid rgba(0, 0, 0, .15);
-  border-width: 1px 0;
-  box-shadow: inset 0 .5em 1.5em rgba(0, 0, 0, .1), inset 0 .125em .5em rgba(0, 0, 0, .15);
-}
-
-.bi {
-  vertical-align: -.125em;
-  fill: currentColor;
-}
-
 .opacity-50 { opacity: .5; }
 .opacity-75 { opacity: .75; }
```

Comment:
I guess those should be dropped since utilities exist already. Maybe another PR later to refactor such styles?

Reply:
Yes I'm going to create another branch + PR to try to refactor it :ok_hand: